terraform/eks/otlp.tf

```diff
@@ -291,6 +293,42 @@ resource "null_resource" "aoc_deployment_adot_operator" {
   depends_on = [module.adot_operator]
 }
 
+data "template_file" "java_auto_instrumentation_config_file" {
```

Comment:
I think you can use the `templatefile` function instead of this data module. https://developer.hashicorp.com/terraform/language/functions/templatefile

Documentation suggest that the function should be used instead of the module. https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file.html

Reply:
thanks. updated accordingly.